mautic/mautic
# Mailjet integration

Please answer the following questions. 

| Q | A |
| --- | --- |
| Bug fix? | N |
| New feature? | Y |
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/86. |
| Related developer documentation PR URL | N |
| Issues addressed (#s or URLs) | N |
| BC breaks? | N |
| Deprecations? | N |

**Note that all new features should have a related user and/or developer documentation PR in their respective repositories.** 
### Required
#### Description:

The PR add mailjet as SMTP relay email router, and manage bounce / spam / blocked status callback.
#### Steps to test this PR:
1. Install this PR
2. Configuration > emails > selected mailjet in the drop down list
3. Add your mailjet login and password.
4. Login to your Mailjet account and go to My Account -> Event tracking (triggers)
5. On the event type list, select the one you want to link to your Mautic account
6. Supports the following webhooks: Message is Bounced, Message is Blocked, Message is Spam.
7. Fill in the webhook URL as `http://your-mautic.com/mailer/mailjet/callback`
8. Send a bounce email and check the mail was routed by Mailjet and declared as bounce.
